README.md

```diff
@@ -1,32 +1,88 @@
-# Superformula Mobile Developer Coding Test
+# Superformula Mobile Test
 
-Make sure you read **all** of this document carefully, and follow the guidelines in it.
+[![Conventional Commits](https://img.shields.io/badge/Conventional%20Commits-1.0.0-green.svg)](https://conventionalcommits.org) 
```

Comment:
Thanks. Since I discovered the **Conventional Commits** approach (about 3 years ago), I instantly fell in love with it. It is so nice to see that you also use this approach.